src/index.ts

```diff
@@ -78,7 +78,7 @@ export default class I18n {
     const num = opts.count
 
     if (typeof num !== 'number') {
-      throw new Error('You must have a `count` property')
+      throw new Error('You must have a `count` property and it cannot be null')
```

Comment:
The code checks that it is a number. "Cannot be null" does not include undefined, strings, booleans, etc. Maybe we should say `and it has to be a number`?

Reply:
@rubenmoya updated! Can you re-check?

Reply:
🚀